### PR TITLE
chore: bump dsp-api (DSP-512)

### PR DESCRIPTION
**Pull request by subotic**

### CODE REVIEW

**Comment by subotic** [comment]
@kilchenmann I've probably committed too much. Could you please check?

**Comment by subotic** [comment]
@kilchenmann I've removed the files as discussed.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -71,6 +73,7 @@ jobs:
           registry-url: https://registry.npmjs.org/
       - name: Install dependencies
         run: |
+          sudo npm install --global @bazel/bazelisk
```
Comment (kilchenmann) [inline .github/workflows/main.yml:76]: Why is this command twice? It's already on line 45

File: Makefile [hunk 2]
```diff
@@ -101,6 +100,9 @@ prepare-test-data: ## prepares test data from knora-api
 	@$(MAKE) -f $(THIS_FILE) generate-test-data
 	@$(MAKE) -f $(THIS_FILE) integrate-test-data
 
+.PHONY: test
+test: npm-install prepare-test-ci prepare-test-ci prepare-test-data unit-tests build e2e-tests ## runs all preparations and tests
```
Comment (kilchenmann) [inline Makefile:104]: `prepare-test-ci` is twice

File: .github/workflows/main.yml (hunk 1 above)
Comment (subotic) [reply .github/workflows/main.yml:76]: this is the second (publish) job

File: Makefile (hunk 2 above)
Comment (subotic) [reply Makefile]: thanks. fixed.

File: .github/workflows/main.yml (hunk 1 above)
Comment (kilchenmann) [reply .github/workflows/main.yml:76]: aha and it's separated from the other one. I understand

Comment (kilchenmann) [reply .github/workflows/main.yml:76]: I'm sorry, I didn't saw the whole context

**Comment by subotic** [comment]
@kilchenmann thanks for the review :-)

**Pull request merged**